index.html

```diff
@@ -0,0 +1,1117 @@
+              </div>
+              <div class="col-lg-12 d-flex list-item latest-news border-top mb-2">
+                <img
+                  src="https://d.newsweek.com/en/full/1054683/rtxw977.webp?w=95&h=95&f=75361f019807cf6d100488e714bd94ed"
+                  alt="" class="w-100 h-100 col-md-3 col-lg-4">
+                <div class="col-lg-8">
+                  <h5>Iceland's Answer To 'Keeping Up With The Kardashians' Has 100% More Cats</h5>
+                </div>
+              </div>
+              <div class="col-lg-12 d-flex list-item latest-news border-top">
+                <img
+                  src="https://d.newsweek.com/en/full/1576072/ohare-airport-coronavirus-travelers-airlines-tsa-emergency.webp?w=95&h=95&f=4e72bda0a3c251f1c462696343a6fd77"
+                  alt="" class="w-100 h-100 col-md-3 col-lg-4">
+                <div class="col-lg-8">
+                  <h5>Hundreds of Thousands of Americans Still Flying Every Day Despite COVID-19</h5>
+                </div>
+              </div>
+              <div class="col-lg-12 d-flex list-item latest-news border-top">
+                <img
+                  src="https://d.newsweek.com/en/full/1576083/andres-manuel-lopez-obrador.webp?w=95&h=95&f=06f88959d55ee595b4228bd9e0e3a2e0"
+                  alt="" class="w-100 h-100 col-md-3 col-lg-4">
+                <div class="col-lg-8">
+                  <h5>Boeing, Airlines, Stoke Controversy With Bailout Pleas for at Least $110bn</h5>
+                </div>
+              </div>
+              <div class="col-lg-12 d-flex list-item latest-news border-top">
+                <img
+                  src="https://d.newsweek.com/en/full/1576071/work-home.webp?w=95&h=95&f=0c3f1b49f71f6e8ed51f5572549308f3"
+                  alt="" class="w-100 h-100 col-md-3 col-lg-4">
+                <div class="col-lg-8">
+                  <h5>We Asked Twitter For Tips To Help Co-Working Partners Not Kill Each Other</h5>
+                </div>
+              </div>
+            </section>
+            <div class="col-8 sign-up-section p-3  offset-lg-1">
+              <div class="d-flex align-items-center pb-3">
+                <div class="icon col-1"></div>
+                <span class="col-9">SIGN UP FOR OUR NEWSLETTER</span>
+              </div>
+              <button class="sign-up-btn p-2 rounded-0">SIGN UP ></button>
+              <p class="pt-3">Update your preferences »</p>
+            </div>
+          </section>
+        </section>
+      </div>
+
+    </div>
+    <div class="row mt-md-5">
+      <div class="row col-lg-12 magazine justify-content-center mt-lg-5 border-bottom">
+        <div class="icon"></div>
+        <h2 class="mt-2 ml-2">in the magazine</h2>
+      </div>
+      <div class="col-md-3 col-lg-3 mt-5">
+        <img
+          src="https://d.newsweek.com/en/full/1575272/fe-covid-cure-opener.webp?w=466&h=311&l=47&t=40&q=90&f=7d5d8ed57150fd6695e0e7d465be9cf5"
+          alt="" class="w-100">
```

Comment:
The first image of this section should be taller than the rest of the images. Please fix this,